ui/apps/everest/.e2e/release/psmdb-sharding.e2e.ts

```diff
@@ -95,20 +95,26 @@ test.describe(
     });
 
     test.afterAll(async ({ request }) => {
-      // we try to delete all monitoring instances because cluster creation expects that none exist
-      // (monitoring instance is added in the form where the warning that none exist is visible)
-      const monitoringInstances = await listMonitoringInstances(
-        request,
-        namespace,
-        token
-      );
-      for (const instance of monitoringInstances) {
-        await deleteMonitoringInstance(
+      // Playwright decided to execute only afterAll hook even if the group is skipped so we need a condition here
+      if (
+        (SELECT_DB ? SELECT_DB === db : true) &&
```

Comment:
can we maybe refactor these conditions into a function, given we use it multiple times?